lighty-applications/lighty-rnc-app-aggregator/lighty-rnc-module/pom.xml

```diff
@@ -40,7 +40,11 @@
         <dependency>
             <groupId>io.lighty.modules</groupId>
             <artifactId>lighty-swagger</artifactId>
-        </dependency>        
+        </dependency>
+        <dependency>
+            <groupId>org.opendaylight.mdsal.binding.model.ietf</groupId>
+            <artifactId>rfc8342-ietf-origin</artifactId>
```

Comment:
Please remove this dependency if it is not needed here. I was able to build an application without it. If it is required, move it to module which use it. This is example and should be here only dependencies required to run lighty example.

Reply:
removed